components/Critical/NavBar.tsx

```diff
@@ -48,6 +69,20 @@ function NavBar() {
             <Typography style={{ color: 'white', fontSize: 16 }}>
               Need help?
             </Typography>
+            {status === 'authenticated' && (
+              <SignOutButton
+                onClick={async () => {
+                  const confirm = window.confirm(
```

Comment:
nothing major just a styling preference -- worth putting this into its own function called signout and then having onClick={signout}?